test/evergreen.yml

```diff
@@ -1930,7 +1930,7 @@ tasks:
           permissions: public-read
           content_type: text/html
           display_name: Coverage report
-          remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
+          remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}_${execution}.html
```

Comment:
This line of change won't take effect until the `coverage-report` task is re-enabled.